2-mandatory/4-stocks.js

```diff
@@ -34,9 +34,24 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  let newAverage = [];
+  let sum = 0;
+  for (let i = 0; i < CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS.length; i++) {
```

Comment:
You should be using the `closingPricesForAllStocks` variable thats passed into the function instead of the globally declared `CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS`  variable. 

The only reason this passes the `npm run test` is because the npm run tests pass in the globally defined one to the function. Any other use case would fail.

Reply:
Noted, will keep that in mind.